packages/stuffex-frontend/src/Item.js

```diff
@@ -80,16 +80,39 @@ function SearchBar(props) {
   const searchBarInput = useRef(null)
 
   const handleSearch = () => props.updateItems(searchBarInput.current.value)
-
+  const restoreItems = () => {
+    searchBarInput.current.value = ''; // Clear the input value
+    props.updateItems(); 
+  };
```

Comment:
Nit: `npm run format` should remove semicolons

Reply:
Thanks, I'll be sure to run this for future commits.